Bot.py

```diff
@@ -23,11 +24,9 @@ class Bot:
 
     # Consts
     CALL_TEXT = "{} second encrypted call at {}"
+    NAMES_TEXT = "{}: {}"
```

Comment:
I currently have this set to:
```suggestion
    NAMES_TEXT = "#{}: {}"
```
Just to make it clear the first part is the badge number

Reply:
I don't really like the idea of using pound here because it will get picked up as a hashtag. maybe `"S/N {}: {}"`?

Reply:
It looks like hashtags that are all numbers don't actually show up! https://help.twitter.com/en/using-twitter/replies-not-showing-up-and-hashtag-problems
(and per the example we have now)